lib/steps_components/text/TextStep.jsx

```diff
@@ -76,7 +76,7 @@ class TextStep extends Component {
     const showAvatar = user ? !hideUserAvatar : !hideBotAvatar;
 
     return (
-      <TextStepContainer className="rsc-ts" user={user}>
+      <TextStepContainer className={`rsc-ts ${user ? 'rsc-ts-user' : null}`} user={user}>
```

Comment:
please change it to `rsc-ts ${user ? 'rsc-ts-user' : 'rsc-ts-bot'}`

Reply:
You are right. Done.

Reply:
HI @LucasBassetti when this one can be merged ?
I need it desperetelly :)